spec/ttt/core_spec.clj

```diff
@@ -14,34 +14,46 @@
                 :players [{:marker "O"} {:marker "X"}]
                 :current-player {:marker "O"}})
 
+(def state-after-two-moves {:board [0 1 2 3 "X" 5 6 7 "O"]
+                            :players [{:marker "X"} {:marker "O"}]
+                            :current-player {:marker "X"}})
+
 (def players [{:marker "O"} {:marker "X"}])
 
 
-(describe "Game"        
-  (context "#set-game-state"
-    (it "sets a new game state"
-      (should= {:board ["X" 1 2 3 4 5 6 7 8], :players players :current-player {:marker "O"}}
-        (game/set-game-state current-state ["X" 1 2 3 4 5 6 7 8] players))))
-  
+(describe "Game"          
   (context "#play"
     (with-stubs)
     
     (it "calls game over message when the game is over"
       (with-redefs [rules/game-over? (stub :rules/game-over? {:return true})]
         
       (should-invoke ui/print-game-over-message {:with []} (game/play current-state))))     
+    
     (it "makes a move when the game is not over"
       (with-redefs [player/get-move (stub :player/get-move {:return 4})
                     game/set-game-state (stub :game/set-game-state {:return new-state})
                     rules/game-over? (let [results (atom [false true])]
                                         (fn [_] (ffirst (swap-vals! results rest))))]
 
       (should-invoke board/make-move {:with [[0 1 2 3 4 5 6 7 8] 4 "X"]} (game/play current-state))))
+    
     (it "prints the new board after a move is made"
       (with-redefs [player/get-move (stub :player/get-move {:return 1})
                     board/make-move (stub :board/make-move {:return [0 "X" 2 3 4 5 6 7 8]})
                     rules/game-over? (let [results (atom [false true])]
                                         (fn [_] (ffirst (swap-vals! results rest))))]
         
       (should= "\n0 | 1 | 2\n---------\n3 | 4 | 5\n---------\n6 | 7 | 8\n\n0 | X | 2\n---------\n3 | 4 | 5\n---------\n6 | 7 | 8\nGame over\n" 
-               (with-out-str (game/play current-state)))))))
+               (with-out-str (game/play current-state)))))
+    
+    (it "prints new boards after each player makes a move"
+      (with-redefs [player/get-move (let [results (atom [4 8])]
+                                      (fn [] (ffirst (swap-vals! results rest))))
+                    board/make-move (let [results (atom [[0 1 2 3 "X" 5 6 7 8] [0 1 2 3 "X" 5 6 7 "O"]])]
+                                      (fn [_a _b _c] (ffirst (swap-vals! results rest))))
```

Comment:
This is a good example of a convention often used in functional programming: putting `_` in front of a variable you don't intend to use.  Also a good example of how `!` typically indicates a function that mutates state or that has a side-effect.

Nothing actionable here - just reinforcing what has probably already come up in your learning over the past couple of weeks.  Nice work on the new tests BTW! 👍